app/assets/stylesheets/hera/modules.scss

```diff
@@ -110,10 +111,6 @@
   width: 0;
 }
 
-.fa-fw {
-  margin-right: 0.25rem;
```

Comment:
This should be a base branch change since it's affecting a global class, not something nested under the node tree sidebar specifically.

This type of declaration doesn't belong under `fa-fw` since we won't always want margin-right on every icon. It caused issues here so I removed this.